qemu/deps/softlockup/heartbeat_slu.py

```diff
@@ -15,7 +15,7 @@ def daemonize(output_file):
     try:
         pid = os.fork()
     except OSError as e:
-        raise Exception("error %d: %s" % (e.strerror, e.errno))
+        raise Exception("error %s: %s" % (e.strerror, e.errno))
```

Comment:
According to the former, looks need to swap the two arguments as well.

Reply:
Yes, it is good to swap their positions.